docs/.vuepress/config.js

```diff
@@ -1,28 +1,80 @@
+const { translate } = require('./theme/utils/translations');
+
 module.exports = {
   title: 'Ethereum',
   description: 'Ethereum resources',
   themeConfig: {
     nav: [
-      { text: 'Ethereum', link: '/' },
-      { text: 'Beginners', link: '/beginners/' },
-      { text: 'Use', link: '/use/' },
-      { text: 'Learn', link: '/learn/' },
-      { text: 'Developers', link: '/developers/' }
+      { text: translate('page-home'), link: '/' },
+      { text: translate('page-beginners'), link: '/beginners/' },
+      { text: translate('page-use'), link: '/use/' },
+      { text: translate('page-learn'), link: '/learn/' },
+      { text: translate('page-developers'), link: '/developers/' }
     ]
   },
   head: [
-    ['meta', {name: 'viewport', content: 'width=device-width,initial-scale=1,maximum-scale=1,maximum-scale=1'}],
-    ['link', {rel: 'icon', type: 'image/png', href: '/favicon.png'}],
+    [
+      'meta',
+      {
+        name: 'viewport',
+        content:
+          'width=device-width,initial-scale=1,maximum-scale=1,maximum-scale=1'
+      }
+    ],
+    ['link', { rel: 'icon', type: 'image/png', href: '/favicon.png' }],
     ['meta', { name: 'twitter:site', content: '@Ethereum' }],
     ['meta', { name: 'twitter:creator', content: '@Ethereum' }],
     ['meta', { property: 'og:type', content: 'website' }],
     ['meta', { property: 'og:title', content: 'Ethereum' }],
     ['meta', { property: 'og:site_name', content: 'ethereum.org' }],
-    ['meta', { property: 'og:description', content: 'Ethereum is a global, decentralized platform for money and new kinds of applications. On Ethereum, you can write code that controls money, and build applications accessible anywhere in the world.'}],
+    [
+      'meta',
+      {
+        property: 'og:description',
+        content:
+          'Ethereum is a global, decentralized platform for money and new kinds of applications. On Ethereum, you can write code that controls money, and build applications accessible anywhere in the world.'
+      }
+    ],
     ['meta', { property: 'og:url', content: 'https://ethereum.org' }],
-    ['meta', { property: 'og:image', content: 'https://ethereum.org/assets/img/hero.bc77fa26.jpg'}]
+    [
+      'meta',
+      {
+        property: 'og:image',
+        content: 'https://ethereum.org/assets/img/hero.bc77fa26.jpg' // TODO this 404s
```

Comment:
@alancwoo I noticed this 404s. Given that the new homepage hero is now an mp4, we should add a static image for this `og:image` property, which is used to generate preview images for social shares. Do you have one handy? If not, I could just take a screenshot of the mp4?